src/components/If/index.js

```diff
@@ -0,0 +1,10 @@
+type IfProps = {
+  condition: boolean,
+  children: any,
+};
+
+/**
+ * Checks if condition passes and renders React children
+ */
```

Comment:
nice component! maybe we could have an `else` prop renders jsx passed in it if the conditions are not met. then the default prop for `else` would be null? Thoughts?

Reply:
sounds good in theory, but i'd rather have 'children' as the renderable rather than having render props... thats the direction i think we should be heading

also:
```
<If condition={true}>
    render this when true...
</If>
<If condition={!true}>
    render this when false...
</If>
```

Reply:
An idea for later-
```
<Condition>
   <If condition={false}>
      <div> I will not render! </div>
   </If>
   <Else>
      <div> I will render! </div>
   </Else>
</Condition>
```